Updated instructions for setting up Stratum 1 (#84)
EESSI/docs: docs/filesystem_layer/stratum1.md

ansible-galaxy role install -r requirements.yml -p ./roles --force
```

Make sure you have enough space in `/srv` (on the Stratum 1) since the snapshot of the Stratum 0
will end up there by default. To alter the directory where the snapshot gets copied to you can add

> this variable in `inventory/host_vars/stratum1host`:

Reviewer: The name of the file should be the same as the host/ip that you use in your hosts file (otherwise Ansible won't pick it up), so maybe it's better to use `<url-or-ip-to-your-stratum1>` here too:
```suggestion
this variable in `inventory/host_vars/<url-or-ip-to-your-stratum1>`:
```

> can edit this variable in `roles/galaxyproject.cvmfs/defaults/main.yml`:

Reviewer: Better not to edit the files from that role, but you can override this variable per host in the `inventory/host_vars` folder (or for all your hosts in `inventory/local_site_specific_vars.yml`, or per group in `inventory/group_vars`, but I guess you often want to do this per host).

Author: Thanks for clarifying!

```bash
cvmfs_srv_mount: /srv
```

Make sure that you have added the hostname or IP address of your server to the

> `/inventory/hosts` file. Finally install the Stratum 1 using one of the two following options.

Reviewer: ```suggestion
`inventory/hosts` file. Finally, install the Stratum 1 using one of the two following options.
```

Make sure to include the IP address of your server.

## Step 3: Verification of the Stratum 1

When the playbook has finished your Stratum 1 should be ready. In order to test your Stratum 1, even

> without a client installed, you can use curl.

Reviewer: ```suggestion
without a client installed, you can use `curl`.
```

```bash
curl --head http://url-to-your-stratum1/cvmfs/pilot.eessi-hpc.org/.cvmfspublished
```
This should return:

```bash
HTTP/1.1 200 OK
...

> X-Cache: MISS from url-to-your-proxy

Reviewer: This is actually the stratum 1's Squid proxy (might be different when you pass `--proxy <your-proxy>` to `curl` as well), so maybe we should use `url-to-your-stratum1` instead.

Author: I thought about that. We both use `url-to-your-stratum1` and `<S1_IP_>`. Should be maybe keep to one? or somehow catch both..

Reviewer: Ah, yes, better to keep that consistent across the page.

Author: is `url-or-ip-to-your-stratum1` too much?

Reviewer: Okay with me :-)

```

The second time you run it, you should get a cache hit:

```bash

> X-Cache: HIT from url-to-your-proxy

Reviewer: Same here

```

Example with the Norwegian Stratum 1:

```bash
curl --head http://bgo-no.stratum1.cvmfs.eessi-infra.org/cvmfs/pilot.eessi-hpc.org/.cvmfspublished
```

> You can also test access to your Stratum 1 from a client
> 
> To test it you will have to install the CMVFS
> [`client`](https://github.com/EESSI/filesystem-layer#clients).

Reviewer: ```suggestion
You can also test access to your Stratum 1 from a client, for which you will have to install the CVMFS
[client](https://github.com/EESSI/filesystem-layer#clients). 
```

Reviewer: This should be moved to the documentation as well, but for now this link is okay. We can change that later on.

> Then you run this commands to add you newly created Stratum 1:

Reviewer: ```suggestion
Then run the following command to add your newly created Stratum 1 to the existing list of EESSI Stratum 1 servers by creating a local CVMFS configuration file:
```

```bash
echo 'CVMFS_SERVER_URL="http://<S1_IP>/cvmfs/@fqrn@;$CVMFS_SERVER_URL"' | sudo tee -a /etc/cvmfs/domain.d/eessi-hpc.org.local
```

If this is the first time you set up the client you now run:

```bash
sudo cvmfs_config setup
```

> If you already have configured a client from before you reload the config:

Reviewer: ```suggestion
If you already had configured the client before, you can simply reload the config:
```

```bash
sudo cvmfs_config reload -c pilot.eessi-hpc.org
```

> Finally you [`verify`](https://github.com/EESSI/filesystem-layer#client) that
> the client can connect to your Stratum 1.
> 
> To check which Stratum 1 you are connected to, run this command:

Reviewer: ```suggestion
Finally, verify that the client connects to your new Stratum 1 by running:
```

```bash
cvmfs_config stat -v pilot.eessi-hpc.org
```

> This should return:

Reviewer: ```suggestion
Assuming that your new Stratum 1 is the geographically closest one to your client, this should return:
```
